packages/medusa-plugin-meilisearch/tsconfig.json

```diff
@@ -22,9 +22,9 @@
       "skipLibCheck": true,
       "downlevelIteration": true // to use ES5 specific tooling
     },
-    "include": ["src"],
+    "include": ["./src/**/*", "index.d.ts"],
```

Comment:
Todo: replace by src, there is no index at the root to add. The root index does not need to exists anymore